src/dateUtils.js

```diff
@@ -17,7 +17,7 @@ export function getLang() {
 
 export function getUserDateFormat() {
   if (typeof ProcessMaker !== 'undefined' && ProcessMaker.user) {
-    return ProcessMaker.user.datetime_format.replace(/ |H|:|m|s|z|Z/g, '');
+    return ProcessMaker.user.datetime_format.replace(/ |H|h|:|m|s|a|A|z|Z/g, '');
```

Comment:
Could this be replaced with a character class instead, something like:
```suggestion
    return ProcessMaker.user.datetime_format.replace(/[\sHh]:[msaAzZ]/g, '');
```
? That should match a single character inside each of the character classes.